.eslintrc.cjs

```diff
@@ -97,14 +97,14 @@ module.exports = {
     'jest/no-conditional-expect': 'off',
 
     // Node rules:
-    'node/no-unsupported-features/es-syntax': [
+    'n/no-unsupported-features/es-syntax': [
       'error',
       {
         ignores: ['dynamicImport', 'modules'], // False positives: https://github.com/mysticatea/eslint-plugin-node/issues/250
```

Comment:
this may be resolved with the new plugin, not sure, but left it alone because I don't have the time.